### DISCUSSION TITLE
Cache time to live by Entity

### DISCUSSION THREAD

**Discussion by theRaffe** [salesforce/akita#721] [Ideas]
```

[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[x] Support request
[ ] Other... Please describe:
```
**Current Behavior**

Im using a EntityStore with ttl property as cache for an API call , each API call is an entity, every time I set an entity the ttl restart 

**Expected Behavior**

I would like to configure a ttl for each entity and invalidate only the old entities

**Comment by NetanelBasal** [comment]
The TTL is per store, and not per entity.
